public/src/reducers/calls.jsx

```diff
@@ -0,0 +1,59 @@
+const defaultCalls = {
+  current_call: undefined,
+  next_call: undefined,
+  calls_made: 0
+};
+
+export const SET_CALL_CURRENT = 'SET_CALL_CURRENT';
+export const SET_CALL_NEXT = 'SET_CALL_NEXT';
+export const INCREMENT_CALLS = 'INCREMENT_CALLS';
+export const CLEAR_CALL_CURRENT = 'CLEAR_CALL_CURRENT';
+export const CLEAR_NEXT_CALL = 'CLEAR_NEXT_CALL';
+export const CLEAR_INC_CALLS = 'CLEAR_INC_CALLS';
```

Comment:
Take it or leave it, but INC is not very clear to me - is it incomplete, incoming, inc (as in company)? I'm wondering if we can be more explicit. Could be just me!

Reply:
INC = increment
I'll change it to clear call count